starter-code/french-onion-soup-starter-code/css/main.css

```diff
@@ -1,9 +1,52 @@
 /*Some necessary adjustments*/
 h1, body, html, main {
   margin: 0;
+  font-size:1em;
 }
 /*Import the font and set the default size*/
 html {
   font-family: 'Open Sans', sans-serif;
-  font-size: 18px
+  font-weight:300;
 }
+header{
+  background-image:url(../images/soup.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: bottom left;
```

Comment:
No habéis probado usar un background-position: center? No entiendo la necesidad del bottom left pero viene bien practicar todas las propiedades, good job 👍